ocs_ci/deployment/deployment.py

```diff
@@ -615,9 +615,9 @@ def do_deploy_hosted_clusters(self):
         """
         Deploy Hosted cluster(s)
         """
-        if (
-            config.ENV_DATA.get("clusters", False)
-            and not config.ENV_DATA["skip_ocs_deployment"]
+        if config.ENV_DATA.get("clusters", False) and (
+            not config.ENV_DATA["skip_ocs_deployment"]
+            or config.DEPLOYMENT.get("deploy_hosted_clusters")
```

Comment:
Can we document this value: deploy_hosted_clusters ?

In config/readme.md file.

Reply:
Updated [623231f](https://github.com/red-hat-storage/ocs-ci/pull/10547/commits/623231f36fc2aebe5d7e9339d1ad5f1c9716f092)